en/advanced-gimbal-control.md

```diff
@@ -33,7 +37,23 @@ O:      10000  10000      0 -10000  10000
 S: 2 2  10000  10000      0 -10000  10000
 ```
 
-Add those you need to your main or auxiliary mixer.
+Add those you need to your [main or auxiliary mixer](advanced-system-startup.md#starting-a-custom-mixer).
+
+There is also a generic quad airframe config which includes a mount mixer:
```

Comment:
I don't yet understand where and how mixers work, but it seems to me that it would be better to somehow link to where the "Generic Quadrotor X config with mount" actually defines the mixer (perhaps in github tree) than to the definition of the airframe.